base/ltmath.dtx

```diff
@@ -754,6 +754,45 @@
 % \end{macro}
 % \end{macro}
 %
+%
+%
+%  \begin{macro}{\dollardollar@begin,\dollardollar@end}
+%    Two commands for starting and ending display math formulas. They
+%    are needed for tagging support and should be used by all display
+%    math environments if they want to be supported by tagging. We
```

Comment:
Better:
. . . environments that need to be . . .

Reply:
sorry, don't understand that suggestion. How about

%  They
%    are needed for tagging support and should be used by all display
%    math environments so that these environments can be used when accessible PDFs
%    are produced.